compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -14572,6 +14573,45 @@ private STNode parseXMLNamePatternChain(STNode startToken) {
         return STNodeFactory.createXMLNamePatternChainingNode(startToken, xmlNamePattern, gtToken);
     }
 
+    /**
+     * Parse xml step extend chain.
```

Comment:
"Chain" is from previous grammar terms I guess. Let's have the exact name from the current spec.
```suggestion
     * Parse xml step extend.
```